providers/dns/inwx/inwx.go

```diff
@@ -152,6 +173,23 @@ func (d *DNSProvider) CleanUp(domain, token, keyAuth string) error {
 		}
 	}()
 
+	if info.TFA == "GOOGLE-AUTH" {
+		if d.config.SharedSecret == "" {
+			return errors.New("two factor authentication but no shared secret is given")
+		}
+
+		var tan string
+		tan, err = totp.GenerateCode("SHARED_SECRET", time.Now())
```

Comment:
This should be `totp.GenerateCode(d.config.SharedSecret, time.Now())`, shouldn't it?

Reply:
Also, the 2fa code block here is identical to the one in `Present()`, we may want to move it into a separate function.